src/System.Security.Cryptography.OpenSsl/ref/System.Security.Cryptography.OpenSsl.cs

```diff
@@ -20,29 +20,9 @@ public sealed partial class DSAOpenSsl : System.Security.Cryptography.DSA
         protected override void Dispose(bool disposing) { }
         public System.Security.Cryptography.SafeEvpPKeyHandle DuplicateKeyHandle() { throw null; }
         public override System.Security.Cryptography.DSAParameters ExportParameters(bool includePrivateParameters) { throw null; }
-        protected override byte[] HashData(byte[] data, int offset, int count, System.Security.Cryptography.HashAlgorithmName hashAlgorithm) { throw null; }
```

Comment:
Did you still need to delete these?

Reply:
Yes, I had to separate it in order to match the ref that we  have today in netstandard2.0. All of these are new members post netstandard2.0, so if I keep them, I would be exposing more API than was available before with this change so I didn't want to do that.